Source/CombatExtended/CombatExtended/Verbs/Verb_LaunchProjectileCE.cs

```diff
@@ -969,43 +1046,16 @@ public override bool TryCastShot()
             // 6:       Interruptible -> stop shooting
             // 7:       Not interruptible -> shoot along previous line
             // 8:     else -> stop
```

Comment:
nit: these should move into the newly extracted method

Reply:
done